flake.nix

```diff
@@ -516,7 +516,7 @@
           hci-effects = hercules-ci-effects.lib.withPkgs pkgs;
         in
         {
-          benchmark-diff = hci-effects.runIf (src.ref == "refs/heads/staging") (
+          benchmark-diff = hci-effects.runIf (src.ref != "refs/heads/master") (
```

Comment:
```suggestion
          # Hercules 0.9 will allow us to calculate the merge-base so we can test all PRs.
          # Right now we just hardcode this effect to test every commit against
          # origin/staging. We set != "refs/head/master" so that merges into master don't
          # cause a lot of unnecessary bogus benchmarks to appear in CI for the time
          # being.
          benchmark-diff = hci-effects.runIf (src.ref != "refs/heads/master") (
```

Reply:
Merge and flatten this, and I think this is a fine approach and we can merge it.

Reply:
I was wondering about that check, and this comment makes it clear to me. 👍🏿

Reply:
Perfect!